src/components/manage/Blocks/Table/CellSidebar.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { BlockDataForm } from '@plone/volto/components';
+import { CellSchema } from './schema';
+import { useIntl } from 'react-intl';
+
+const CellSidebar = (props) => {
```

Comment:
@danielamormocea We don't require two BlockDataForms. You can directly add multiple fieldsets (which have their own schema ) like in https://github.com/eea/volto-accordion-block/blob/master/src/components/manage/Blocks/Accordion/Schema.js#L72